app/src/main/java/com/kickstarter/libs/rx/transformers/Transformers.java

```diff
@@ -176,6 +176,9 @@ private Transformers() {}
   /**
    * If called on the main thread, schedule the work immediately. Otherwise delay execution of the work by adding it
    * to a message queue, where it will be executed on the main thread.
+   *
+   * This is useful for `RecyclerView`s where new data can appear immediately rather than waiting for a frame
```

Comment:
I might slightly rephrase this –

> This is particularly useful for `RecyclerView`s; if subscriptions in these views are delayed for a frame, then the views temporarily shows recycled content and frame rate stutters. To address that, we can use `observeForUI` to execute the work immediately rather than wait for a frame.

PR 230 in the old repo goes into more detail.